### ISSUE TITLE
Ramdrive not showing up

### ISSUE THREAD

**Issue by Hbitvof** [MrPig91/PSChiaPlotter#166]
I use a ramdrive of 120 GB for the plotting process but I can't select this as tempdrive. Can this be fixed?

**Comment by MrPig91** [comment]
As of right now I am not working on a fix for this but you should be able to use it using the basic plotting section where you can give a full path to the temp folder. Sorry for the inconvenience.

**Comment by Hbitvof** [comment]
> As of right now I am not working on a fix for this but you should be able to use it using the basic plotting section where you can give a full path to the temp folder. Sorry for the inconvenience.

I don't understand your answer. When loading the GUI (show-pschiaplotter) there is no option to select the ramdrive (this is just a driveletter). This driveletter is not available in the GUI.
